modules/sim_2d/sim_2d_component.cc

```diff
@@ -293,28 +272,26 @@ void SimControl::OnPlanning(const std::shared_ptr<ADCTrajectory>& trajectory) {
   }
 }
 
-void SimControl::Freeze() {
+void Sim2DComponent::Freeze() {
   next_point_.set_v(0.0);
   next_point_.set_a(0.0);
   prev_point_ = next_point_;
 }
 
-void SimControl::RunOnce() {
+void Sim2DComponent::SimStep() {
   std::lock_guard<std::mutex> lock(mutex_);
 
   TrajectoryPoint trajectory_point;
   Chassis::GearPosition gear_position;
   if (!PerfectControlModel(&trajectory_point, &gear_position)) {
     AERROR << "Failed to calculate next point with perfect control model";
-    return;
```

Comment:
How come we don't want to return anymore?

Reply:
Sorry. That was mistake. Bad refactor.